### ISSUE TITLE
Sending scripts to XYplorer run scripts ?

### ISSUE THREAD

**Issue by SheepDomination** [Otiel/vscode-xyplorer#1]
I was wondering whether it can be added whereas, you can send the script to XYplorer > Scripting > Run Script and vise versa ?

**Comment by Otiel** [comment]
There may be a possibility since we can use the following command line to run a script:

    "C:\Program Files (x86)\XYplorer\XYplorer.exe" /feed=Test.xys

That would require some testing to validate the behavior on XYplorer side.
I'm not fluent with VS code extensions authoring, so any help or PR would be appreciated.

What do you mean by "vice-versa" ?

**Comment by SheepDomination** [comment]
What do you mean by PR ?

When I mentioned "vise-versa" I meant; not just sending the script from visual studio code to XYplorer > run script; also sending what code that is within the *run script* command back to visual studio code for further editing.  
Rather then copying and pasting the code into XYplorer *run script*, you would trigger to send the script to XYplorer *run script* from within the visual studio code extension, then if you should so happen to edit the script within XYplorer *run script* you could send that back to the tab which you are working on the XYplorer script within visual studio code.

**Comment by Otiel** [comment]
PR = pull request.

Oh, my bad, I didn't understand what you meant. I thought you wanted to run a script directly from VS code.

What you're looking for (copy-pasting from one window to another window) falls out of VS code extensions scope. You should look for doing this using an automation software, like AutoHotkey or AutoIt.

**Comment by SheepDomination** [comment]
I originally wanted to run a script directly from VSCode, not sure if that is possible ?

**Comment by Otiel** [comment]
I think it is, per what I wrote on my [first comment](https://github.com/Otiel/vscode-xyplorer/issues/1#issuecomment-421359229), but the script will be automatically executed: it won't be sent to the "debug" window of XYplorer.

**Comment by SheepDomination** [comment]
One would always have to re-start XY and feed it the `/feed=Test.xys` flag for the script which you want XY to execute ?

**Comment by Otiel** [comment]
Not at all. The goal would be to create a VS code command that would execute this command line:

    "C:\Program Files (x86)\XYplorer\XYplorer.exe" /feed=Test.xys

That would not restart XYplorer (see the [Help file](https://www.xyplorer.com/download/XYplorerHelp.pdf), chapter "Command Line Switches"); that would only execute the script in the existing XYplorer instance.

**Comment by SheepDomination** [comment]
What VS code command would have to be executed, or is this command what you'd add to the extension ?

**Comment by Otiel** [comment]
>is this command what you'd add to the extension ?

Correct.

**Comment by SheepDomination** [comment]
> > is this command what you'd add to the extension ?
> 
> Correct.

I look forward to the update 👍

**Comment by SheepDomination** [comment]
Any update and for code be sent to the debug window ?

**Comment by Otiel** [comment]
No sorry, I won't have time to work on this in the near future. 🙁

**Comment by SheepDomination** [comment]
If you ever decide to start it up; this is the only feature request I'd like to see ;)
